### ISSUE TITLE
[Bug]: SyntaxError: Unterminated template. (3:16) from @mdx-js/loader/index.js

### ISSUE THREAD

**Issue by laujonat** [phase2/outline#263]
**Describe the bug**
Running `yarn storybook:prod` fails to complete production build.  

**To Reproduce**

1. Fork project
2. Run `yarn storybook:prod`

Command fails on `NODE_ENV=production build-storybook -c src/.storybook -o storybook-static`

**Expected behavior**
Completed build. 

**Screenshots**
```bash
info => Copying static files: /home/tlcie/dev/lib/nub-ui/src/.storybook/static at /home/tlcie/dev/lib/nub-ui/storybook-static/
info => Loading custom manager config
info => Compiling manager..
info => Compiling preview..
info => Loading custom babel config as JS
info => Using PostCSS preset with postcss@8.4.8
info => Using default Webpack5 setup
info => Loading custom manager config
ERR! => Failed to build the preview
ERR! Module build failed (from ./node_modules/@mdx-js/loader/index.js):
ERR! SyntaxError: Unterminated template. (3:16)
ERR!     at Object._raise (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:476:17)
ERR!     at Object.raiseWithData (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:469:17)
ERR!     at Object.raise (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:430:17)
ERR!     at Object.readTemplateToken (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:3153:20)
ERR!     at Object.getTokenFromCode (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:2661:14)
ERR!     at Object.getTokenFromCode (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:7848:18)
ERR!     at Object.nextToken (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:2100:10)
ERR!     at Object.next (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:2001:10)
ERR!     at Object.parseIdentifierName (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:13577:10)
ERR!     at Object.parseIdentifier (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:13544:23)
ERR!     at Object.parseExprAtom (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:12592:27)
ERR!     at Object.parseExprAtom (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:7812:20)
ERR!     at Object.parseExprSubscripts (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:12149:23)
ERR!     at Object.parseUpdate (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:12129:21)
ERR!     at Object.parseMaybeUnary (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:12104:23)
ERR!     at Object.parseMaybeUnaryOrPrivate (/home/tlcie/dev/lib/nub-ui/node_modules/@babel/parser/lib/index.js:11901:61)
ERR! Error: => Webpack failed, learn more with --debug-webpack
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/@storybook/builder-webpack5/dist/cjs/index.js:200:62
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/HookWebpackError.js:68:3
ERR!     at Hook.eval [as callAsync] (eval at create (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
ERR!     at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/Hook.js:18:14)
ERR!     at Cache.shutdown (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Cache.js:150:23)
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:1212:15
ERR!     at Hook.eval [as callAsync] (eval at create (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
ERR!     at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/Hook.js:18:14)
ERR!     at Compiler.close (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:1205:23)
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/@storybook/builder-webpack5/dist/cjs/index.js:199:20
ERR!     at finalCallback (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:441:32)
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:458:13
ERR!     at Hook.eval [as callAsync] (eval at create (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
ERR!     at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/Hook.js:18:14)
ERR!     at onCompiled (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:456:21)
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:1183:17
ERR!  Error: => Webpack failed, learn more with --debug-webpack
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/@storybook/builder-webpack5/dist/cjs/index.js:200:62
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/HookWebpackError.js:68:3
ERR!     at Hook.eval [as callAsync] (eval at create (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
ERR!     at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/Hook.js:18:14)
ERR!     at Cache.shutdown (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Cache.js:150:23)
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:1212:15
ERR!     at Hook.eval [as callAsync] (eval at create (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
ERR!     at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/Hook.js:18:14)
ERR!     at Compiler.close (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:1205:23)
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/@storybook/builder-webpack5/dist/cjs/index.js:199:20
ERR!     at finalCallback (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:441:32)
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:458:13
ERR!     at Hook.eval [as callAsync] (eval at create (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
ERR!     at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/node_modules/tapable/lib/Hook.js:18:14)
ERR!     at onCompiled (/home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:456:21)
ERR!     at /home/tlcie/dev/lib/nub-ui/node_modules/webpack/lib/Compiler.js:1183:17
error Command failed with exit code 1.

```

**Additional context**

Using `node v16.14.0 (npm v8.3.1)`
I've tried using `babel-loader@8.1.0` to no prevail.  The other babel packages look correct too. Unsure whether it is a problem with my environment or with babel.

**Comment by himerus** [comment]
Hi @laujonat ! 

I hate to ask the simplest of questions of start, but did you run `yarn install` prior to `yarn storybook:prod`?

**Comment by shaal** [comment]
Hi @laujonat, welcome 👋 

I tried replicating the error you were seeing, but I wasn't able to.

I use Gitpod (full development environment in the browser)

If you click on this Gitpod button
_(you'll be asked once to authorize Gitpod to access Github)_
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/phase2/outline)

A fresh new copy of Outline gets installed (`yarn install`) and then automatically run storybook (`yarn start`).
You can stop storybook in terminal, and run the command `yarn storybook:prod`

Can you replicate the issue you were seeing in that environment?

**Comment by laujonat** [comment]
Hi thank you both for the replies.  

@himerus yes, I installed the required dependencies. 
@shaal I did a fresh copy of Outline and was able to create the production build successfully as well. 

I think the problem is in the component: `<outline-code-block>`.  

**To Reproduce**
1. Open `outline-code-block.stories.mdx`
2. Within any `html` template literal, add an empty line. 

Running Storybook or running the production build will invoke the error for me:
```
ModuleBuildError: Module build failed (from ./node_modules/@mdx-js/loader/index.js):
SyntaxError: Unterminated template. (3:16)
```

Is this expected behavior with `outline-code-block`? Not sure if `html` from lit is causing it, but it would be nice if we could catch this error because the mdx-js error wasn't very helpful 😄 
I made the same edit in another story that doesn't use a dynamic return and it doesn't throw any errors.

**mdx error**
```
 {() => {
    return html`
      <outline-code-block language="html">
      
      <outline-code-block language="css">
      ...
``` 

**OK**
```
`
docs: {
    source: {
      code: `
<outline-admin-links>
    <ul>
                
         <li>
            <a href="#" class="is-active">
             ...
</outline-admin-links>
``` 

I think we can probably close out this issue though.
